backend/src/main/java/com/yohoard/notes/DevController.java

```diff
@@ -0,0 +1,59 @@
+package com.yohoard.notes;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Controller
+@RequestMapping("/dev")
+public class DevController {
+    private final NoteRepository noteRepository;
+    private final CategoryRepository categoryRepository;
+    private final TagRepository tagRepository;
+    private final HighlightRepository highlightRepository;
+
+    @Autowired
+    public DevController(
+            NoteRepository noteRepository,
+            CategoryRepository categoryRepository,
+            TagRepository tagRepository,
+            HighlightRepository highlightRepository
+    ) {
+        this.noteRepository = noteRepository;
+        this.categoryRepository = categoryRepository;
+        this.tagRepository = tagRepository;
+        this.highlightRepository = highlightRepository;
+    }
+
+    @PostMapping("/fixtures")
```

Comment:
Te fixtury ogólnie nie powinny być w appce, ani pseudokontroler tego typu. Bo to dane, które chcesz mieć załadowane do bazy z automatu przy starcie aplikacji. Niech narazie tak zostanie, ale nie przywiązuj się za mocno do tego kodu, bo to na skrypty init dla mongo trzeba będzie zamienić :P

Reply:
I przy starcie i później w dowolnej chwili jeśli chcielibyśmy cokolwiek dodać żeby na szybko przetestować

Reply:
To nadal dużo bardziej nadają się do tego skrypty i mongo shell, a nie już działająca apka jako punkt w którym ładujesz dane jakich potrzebujesz 🙂 Jeśli miałeś kiedyś do czynienia z relacyjnymi bazami danych to pewnie widziałeś gdzieś init skrypty, które tworzą tabele i ładują dane, takie zwykłe pliczki sql w sumie. Ponieważ mongo jest nosqlową bazą te pliczki nie będą napisane w sql'u, a z tego co guglałam to w js'ie. Ale to kilka prostych funkcji jest, nic strasznego i specjalnie trudnego.

Reply:
Z tą zmianą chodziło mi bardziej o to, że dorzuciłem też do zależności devtoolsy, żeby po zmianach w tej klasie od razu przebudowywało aplikację i można było puszczeniem requesta zobaczyć czy napisany tutaj kod, metody w  repozytoriach itd są poprawne. Taki szkielet który jak sama nazwa mówi jest typowo pod dev

Reply:
Nadal nie bardzo rozumiem cel dla którego chcemy ten kod trzymać 🤔 Bo od sprawdzania czy metody w repozytoriach robią co powinny są testy, a podając dane jakie chcemy załadować do testcontainers to już w ogóle sprawa wydaje się wygodna i prosta z podejrzeniem jakie dane lądują w bazie i czy się one zgadzają. Dodanie devtoolsów w zależnościach by apka miała live reload i automatyczny restart by strzelać tymi danymi ręcznie z endpointa wydaje mi się zupełnie niepotrzebne